pkg/config/sync_test.go

```diff
@@ -235,6 +236,8 @@ func TestSyncConfigDoesNotEraseClusterRoutingSuffix(t *testing.T) {
 
 func TestMergeConfigLooksAtAllFields(t *testing.T) {
 	f := fuzz.New().NilChance(0)
+	// The DefaultTemplate struct contains the nested-struct runtime.RawExtension which has an Object field that causes the fuzzer to panic
+	f.SkipFieldsWithPattern(regexp.MustCompile("^Object$"))
```

Comment:
You're looking for something like
```suggestion
	f := fuzz.New().NilChance(0).Funcs(
	  func(embeddedResource *runtime.RawExtension, c fuzz.Continue) {}
	  func(attributesJSON *apiext.JSON, c fuzz.Continue) {}
	)
```
(i.e. -- don't generate anything for those types)

Reply:
For reference, [this](https://github.com/devfile/api/blob/351f05b7c2b18e8a449dad444029cdf5e1ca81cc/pkg/apis/workspaces/v1alpha1/conversion_test.go#L16-L206) is how we use this sort of thing in devfile/api tests.

Reply:
Yes that's much better, thank you :) I ended up going with only `func(embeddedResource *runtime.RawExtension, c fuzz.Continue) {}` as that was enough to let the test pass. However, is there a reason I should also make the fuzzer skip over the `apiext.JSON`?